lib/uint64.js

```diff
@@ -43,7 +43,7 @@ class Uint64 {
       }
       const uintRadix = new Uint64(radix);
       let res = new Uint64();
-      for (let i = 0; i < value.length; i++) {
+      for (let i = radix !== 2 ? 0 : 2; i < value.length; i++) {
```

Comment:
```suggestion
      for (let i = radix === 10 ? 0 : 2; i < value.length; i++) {
```
?

Reply:
I've done it the way you propose at first, but then I've changed it to the way it is now since it is shorter and results in the same behavior.